src/pages/private/dashboard.js

```diff
@@ -8,15 +10,13 @@ import styles from "./dashboard.module.css";
 
 const Dashboard = (props) => {
   const context = useContext(AuthContext);
```

Comment:
1. Rename the theme context as ThemeContext and seperate it from Auth context
2. Create a common hook to get the theme. Something like:

```
const useTheme = () => useContext(ThemeContext);

// and use it like this on consumers
const theme = useTheme();
```